src/game/objects/general/door.c

```diff
@@ -51,12 +51,12 @@ static void Door_Shut(DOORPOS_DATA *const d)
         return;
     }
 
-    sector->index = 0;
```

Comment:
Can you explain a bit more regarding why this assignment is no longer necessary?

Reply:
This goes hand in hand with the line below.